core/libraries/form_sections/strategies/display/EE_Admin_File_Uploader_Display_Strategy.strategy.php

```diff
@@ -57,7 +57,13 @@ public function display()
         $input .= 'value="' . $this->_input->raw_value_in_form() . '" ';
         $input .= $this->_input->other_html_attributes() . '>';
         // image uploader
-        $uploader = EEH_HTML::link('#', '<img src="' . admin_url('images/media-button-image.gif') . '" >', esc_html__('click to add an image', 'event_espresso'), '', 'ee_media_upload');
+        $uploader = EEH_HTML::link(
+            '#',
+            '<img src="' . admin_url('images/media-button-image.gif') . '" >',
+            esc_html__('click to add an image', 'event_espresso'),
+            '',
+            'ee_media_upload button'
```

Comment:
can you plz change this to:

```suggestion
            'ee_media_upload button button--secondary'
```

so that it picks up the new admin styles in the dev branch?

thnx

Reply:
Done.